zeppelin-zengine/src/main/java/org/apache/zeppelin/interpreter/InterpreterOption.java

```diff
@@ -21,25 +21,25 @@
  *
  */
 public class InterpreterOption {
-  boolean remote;
+//  boolean remote;
```

Comment:
I think it's better keep this field for a while (with mark @deprecated or something)
Without this field exist with 'true' value assigned, when previous version of Zeppelin read notebook created from newer version, previous version of Zeppelin will assume remote = 'false' and some interpreters will not work. (e.g. spark)